Narnavirus subdolus report and video submission

Wang-Clara opened the pull request:
Add virus discovery report, images, and video for Narnavirus subdolus

Wang-Clara committed:
Submitted report and video

Wang-Clara committed:
Remove dotfiles and ignore them

ababaian commented on .gitignore:
Remove your dot files please.

ababaian commented on virus/img/narnavirus_subdolus/viral_genome.png:
Please don't include any whitespace in any filenames. Replace them with underscores and fix the references to those files.

ababaian commented on virus/narnavirus_subdolus.md:
Nice job on this PR :)

ababaian reviewed (changes requested):
Few minor fixes to have PR accepted.

Wang-Clara committed:
Rename image files to remove spaces

Wang-Clara commented:
Hi Dr. Babaian,

I've removed the dot files and removed the .gitignore, and also changed the file names to underscores instead of spaces. Please let me know anything is missing.

Thanks,
Clara

Wang-Clara committed:
Remove .gitignore
